### PR TITLE
EWS: Changed access restriction to input stock data

### PR DESCRIPTION
**Pull request by kkrampa**

### CODE REVIEW

File: custom/ewsghana/views.py [hunk 1]
```diff
@@ -119,7 +119,7 @@ def post(self, request, *args, **kwargs):
                 }
                 process(self.domain, unpacked_data)
             url = make_url(
-                StockLevelsReport,
+                StockStatus,
```
Comment (kkrampa) [inline custom/ewsghana/views.py]: Redirect to StockLevelsReport raises 404 to read only users.

File: custom/ewsghana/tests/test_input_stock_view.py [hunk 2]
```diff
@@ -192,6 +203,17 @@ def test_web_user_with_valid_parent_location_access(self):
...
+    def test_admin_web_user_access(self):
+        self.client.login(username=self.admin_username, password=self.admin_password)
+
+        view_url = reverse('input_stock', kwargs={'domain': TEST_DOMAIN, 'site_code': 'tsactive'})
+        response = self.client.get(view_url, follow=True)
+        self.assertEqual(response.status_code, 200)
+
+        view_url = reverse('input_stock', kwargs={'domain': TEST_DOMAIN, 'site_code': 'rsp'})
+        response = self.client.get(view_url, follow=True)
+        self.assertEqual(response.status_code, 200)
```
Comment (gcapalbo) [inline custom/ewsghana/tests/test_input_stock_view.py]: did it used to be the case that any web user could access the input_stock page, and now only admins can?  if so can we also add a test to make sure that other web users (such as read only web users) don't have access to it anymore

Comment (kkrampa) [inline custom/ewsghana/tests/test_input_stock_view.py]: I forgot to paste link to case: http://manage.dimagi.com/default.asp?190529#1068195.
Mike didn't write that now only admins should have access to input stock data so it seems existing logic should be leaved.

Comment (gcapalbo) [inline custom/ewsghana/tests/test_input_stock_view.py]: oh i see, we're just making it so that admins can input stock for all facilities

**Pull request merged**